src/partials/resource-limits.md

```diff
@@ -0,0 +1,62 @@
+## Check resource usage {% #check-resource-usage %}
+
+You can check your organization's resource usage by navigating to the overview page of your organization. 
```

Comment:
I dont think there is `overview` page. There is `projects`, `usage` and `billing`

Reply:
Fixed